redis/src/aio/connection_manager.rs

```diff
@@ -301,4 +302,9 @@ impl ConnectionLike for ConnectionManager {
     fn get_db(&self) -> i64 {
         self.client.connection_info().redis.db
     }
+
+    fn is_closed(&self) -> bool {
+        // always return false due to automatic reconnect
+        false
```

Comment:
As usual with `state`s, I would rather see them added as `bitflags!` rather than `boolean`. This allows adding future states without increasing the size of the connection and with minimal code changes

Reply:
this function's name (is_) leaves absolutely no option to have something else than boolean

Reply:
This is not what I meant.
I meant: don't keep the state of the connection as a boolean.

You could:

```rust
use bitflags::bitflags;

bitflags! {
    pub struct Flags: u32 {
        const CONNECTED = 0 << 1;
        const ADMIN = 0 << 2;
    }
}
```

and then:

```rust
pub fn is_connected(&self) -> bool {
    self.flags.intersect(Flags::CONNECTED)
}
```

etc

Reply:
I understood, I could but there is no need